packages/server/test/logger.test.ts

```diff
@@ -0,0 +1,25 @@
+import { describe, it } from 'mocha';
+import * as expect from 'expect';
+import { actions } from './helpers';
+import { createLogger } from '../src/logger';
+import { createOrchestratorAtom } from '../src';
+import { OrchestratorState } from '../src/orchestrator/state';
+import { Atom } from '@bigtest/atom/dist';
+import { BundlerState } from '@bigtest/bundler/src';
+
+describe('logger', () => {
+  it('should log bundler events', async () => {
+    let atom: Atom<OrchestratorState>;
+    let logger:  (<A extends unknown[]>(...a: A) => void) = (...args) => {
+      expect(args).toEqual(["[manifest builder] build error:", "blah"]);
+    };
+
+    atom = createOrchestratorAtom();
+    
+    actions.fork(function* () {
+      yield createLogger({ atom, out: logger });
+    })
```

Comment:
again here you can fork the logger directly, no need to wrap it:

```suggestion
    actions.fork(createLogger({ atom, out: logger }));
```